src/collective/importexport/browser/import_view.py

```diff
@@ -636,52 +718,66 @@ def handleImportExport(self, action):
             normalizer = getUtility(IIDNormalizer)
             random_id = normalizer.normalize(time.time())
             filename = "export_{0}.{1}".format(random_id, 'csv')
-            attachment = export_file(self.import_metadata["report"],
+            dataset = export_file(self.import_metadata["report"],
                                                header_mapping,
                                                self.request)
+            file = StringIO.StringIO()
+            file.write(dataset.csv)
+            attachment = file.getvalue()
```

Comment:
Why do we need to write `dataset.csv` to a file and then read it, can't we just use `dataset.csv` directly?